lib/meatauto.gi

```diff
@@ -1681,8 +1681,8 @@ end;
 
 SMTX.BasisModuleEndomorphisms:=SMTX_BasisModuleEndomorphisms;
 
-SMTX.SetBasisEndomorphismsRadical:=SMTX.Setter(MakeImm("basisEndoRad"));
```

Comment:
Elsewhere you dropped the `MakeImmutable`, as you moved them in method.

Reply:
Oops, that happens when you create different parts of a patch at different times. Resolved.